virtcontainers/qemu.go

```diff
@@ -1977,7 +1977,7 @@ func genericBridges(number uint32, machineType string) []types.Bridge {
 	case QemuPC:
 		bt = types.PCI
 	case QemuVirt:
-		bt = types.PCIE
+		bt = types.PCI
```

Comment:
hello @amshinde -, qemuvirt is only for Arm, I correct it here because for now the only bridge created on arm is pcie-pci-bridge which is a pci bridge.

Reply:
@jongwu afaik, qemuvirt is supported on x86_64 as well, so if this is a limitation on arm,  maybe its better to make this change in arm specific code path. cc @devimc

Reply:
Ok nvm, I am good with this change, since we no longer maintain virt machine type for x86.